meta-resin-common/recipes-core/images/resin-image-initramfs.bb

```diff
@@ -6,6 +6,7 @@ with some key operations."
 PACKAGE_INSTALL = " \
     base-passwd \
     busybox \
+    e2fsprogs-e2fsck \
```

Comment:
Why is it needed here? Shouldn't this be a dependency of the initramfs sript?

Reply:
To make it clearer that the package will be in the initramfs. Such as "udev" is listed below. But I will change it as requested